src/js/binaryen.js-post.js

```diff
@@ -84,6 +84,14 @@ Module['ExternalTable'] = Module['_BinaryenExternalTable']();
 Module['ExternalMemory'] = Module['_BinaryenExternalMemory']();
 Module['ExternalGlobal'] = Module['_BinaryenExternalGlobal']();
 
+// Features
+Module['FeatureAtomics'] = Module['_BinaryenFeatureAtomics']();
+Module['FeatureBulkMemory'] = Module['_BinaryenFeatureBulkMemory']();
+Module['FeatureMutableGlobals'] = Module['_BinaryenFeatureMutableGlobals']();
+Module['FeatureNontrappingFPToInt'] = Module['_BinaryenFeatureNontrappingFPToInt']();
+Module['FeatureSignExt'] = Module['_BinaryenFeatureSignExt']();
+Module['FeatureSIMD128'] = Module['_BinaryenFeatureSIMD128']();
```

Comment:
Are these names idiomatic enough?

Reply:
How about an API that looks like `Binaryen.Features.Atomics`?

cc @dcodeIO

Reply:
Sounds good :)